Makefile.toml

```diff
@@ -93,6 +98,9 @@ PUBLISH_REPO_OUTPUT_DIR = "${PUBLISH_REPO_BASE_DIR}/${BUILDSYS_NAME_VERSION}"
 # The default name of registered AMIs; override by setting PUBLISH_AMI_NAME.
 PUBLISH_AMI_NAME_DEFAULT = "${BUILDSYS_NAME}-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-v${BUILDSYS_VERSION_IMAGE}-${BUILDSYS_VERSION_BUILD}"
 
+# Repo directories for the refresh and resign non-root metadata task
+REFRESH_REPO_OUTPUT_DIR = "${PUBLISH_REPO_BASE_DIR}/refreshed-repos"
```

Comment:
I think it'd be better to use the existing PUBLISH_REPO_OUTPUT_DIR, and (like in the `repo` task) fail if that directory exists.  I think people will rarely be doing both of those things in the same checkout, and it's clearer to have one spot for repos.

(After rebasing on #1135, it'll be easier to use the same path, since we can pass in PUBLISH_REPO_OUTPUT_DIR, which now includes the repo name, rather than basing the path on the metadata URL.)